.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -8,4 +8,15 @@
 
 # Checklist
 
-+ [ ] Tested latest version in an Internet Explorer virtual machine? *(If the PR adds Javascript.)*
+*If the PR touches Javascript, follow this checklist. If the PR doesn't touch Javascript, you can delete this checklist.*
+
+*Which features or pages does this PR touch? Make a list here so that you can double-check each affected page in a Internet Explorer before merging. Make one list for the author and another for the reviewer. For more info on how to download and use a Virtual Machine to test in IE, see the README.*
+
+*Examples:*
+
++ [ ] Author checked latest in IE - Student Profile
```

Comment:
Can we write automated tests for any of this?  The bugs mentioned in the description don't have anything to do with IE and would have been caught by any tests that exercised that code.   In particular, I'm wondering if we could write a JS test for the student profile page that renders three data points for each chart, and then verifies there are that many data points on the page.

Also, checklists and manual testing are great too!  I'm just wondering if automated tests could help as well.

Reply:
Yes to more tests! And to checklists. How about an item in the checklist that talks about adding test coverage?

Reply:
When I was working on the refactoring I also notice parts of the app that doesn't have specs for. I was considering adding them. Here is also a vote from me for more tests.

Reply:
@alexsoble @kevinrobinson Have you guys seen `yarn test --coverage`? Lots of js in the red and `app/assets/javascripts/student_profile/pdf` is simply not tested.